received_friend.py

```diff
@@ -0,0 +1,153 @@
+import matplotlib.pyplot as plt
+import os
+import timestring
+import datetime
+from datetime import timedelta
+import dateutil.parser
+import numpy as np
+import json
+from tabulate import tabulate
+
+
+
```

Comment:
Why 3 lines ideally it's 2 and it will be really good if you pass it through pep8, not all rules are good but some formatings can be taken care of.